workers/redirector.js

```diff
@@ -0,0 +1,65 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * This is a custom URL redirector implemented as a CloudFlare Worker.
+ * The role of the worker is to redirect a predefined % of traffic 
+ * to a different URL (used for A/B testing).
+ * API documentation can be found at https://developers.cloudflare.com/workers/
+ */
+
+/**
+ * The origin that the worker is to be run on.
+ * (e.g. https://www.allizom.org, https://www.mozilla.org).
+ */
+const origin = 'https://www-dev.allizom.org';
+
+/**
+ * `targetURL` is the URL that the worker looks to match against.
+ * `sandboxURL` is the experiment URL to redirect to.
+ * `sampleRate` is the proporation of traffic that should be redirected (a value of 0.1 equates to 10%).
+ */
+const experimentPages = [
+    {
+        'targetURL': `${origin}/en-US/firefox/new/`,
+        'sandboxURL': `${origin}/en-US/exp/firefox/new/`,
+        'sampleRate': 0.5
```

Comment:
I believe we wanted to use a lower initial sample rate like 0.05 here.

Reply:
I talked with Justin and I'm going to set this at 5%